xform-marc21-to-xml/src/test/java/edu/stanford/MarcUtils.java

```diff
@@ -0,0 +1,72 @@
+package edu.stanford;
+
+import org.apache.commons.io.FileUtils;
+import org.marc4j.MarcStreamReader;
+import org.marc4j.MarcWriter;
+import org.marc4j.MarcXmlWriter;
+import org.marc4j.marc.Record;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static java.nio.file.Files.createTempDirectory;
+import static java.nio.file.Files.createTempFile;
+
+/**
+ *
+ */
+public class MarcUtils {
```

Comment:
Could this class be renamed to something like `MarcTestUtils` maybe just to make clear that these methods are only used in the context of tests?